src/test_l1_plat_power.c

```diff
@@ -74,6 +74,30 @@
 static int gTestGroup = 1;
 static int gTestID = 1;
 
+#define PLAT_ASSERT_AUTO_TERM_NUMERICAL(value, comparison){\
```

Comment:
What's missing from Version 2 that requires you todo this?

Reply:
UT_ASSERT_FATAL( value == comparison )

Reply:
We also need to do proper termination before aborting the test case using UT_ASSERT_FATAL.  The above API will do proper termination of plat_init before exits.